plugins/foreman_docker/3.x/index.md

```diff
@@ -0,0 +1,311 @@
+
+With Foreman and Katello, you can create an on-premise registry, [import images from various sources and distribute them to containers using content views](http://www.katello.org/docs/2.2/user_guide/docker/docker.html). Foreman supports creating a Docker compute resource, that acts as a server for running containers. This way, you can import an image, start a container based on this image, monitor the container's activity, and commit it's state to a new image layer that can be further propagated.
+
+![Containers list](static/images/plugins/foreman_docker/docker-containers-list.png)
+
+# 3. Installation
+
+## 3.1 Foreman Installer (recommended)
+You can use the Foreman installer to install this plugin. Use the following command:
+    foreman-installer --enable-foreman-plugin-docker
+
+If you prefer not to use the installer, follow the instructions below for your operating system.
+
+## 3.2 Red Hat, CentOS, Fedora, Scientific Linux (rpm)
+
+Set up the repo as explained in the link above, then run
+
+    # yum install ruby193-rubygem-foreman_docker
+
+## 3.3 Debian, Ubuntu (deb)
+
+Set up the repo as explained in the link above, then run
+
+    # apt-get install ruby-foreman-docker
+
+## 3.4 Bundle (gem)
+
+This method will **only** work on Foreman deployments installed from source. Please refrain from making these changes if you have installed Foreman via packages.
+
+Add the following to bundler.d/Gemfile.local.rb in your Foreman installation directory (/usr/share/foreman by default)
+
+    $ gem 'foreman_docker'
+
+Then run `bundle install` and `rake db:migrate` from the same directory
+
+--------------
+
+To verify that the installation was successful, go to Foreman, top bar **Administer > About** and check 'foreman_docker' shows up in the **System Status** menu under the Plugins tab. You should also see a **'Containers'** button show up in the top bar, similar to the one in this image:
+
+![](static/images/plugins/foreman_docker/docker-topbar-check.png)
+
+# 4. Usage
+
+## 4.1 Managing containers
+
+The following sections show how to create, view, start, stop, and commit a container.
+
+### ⁠Prerequisites
+
+In Foreman, you can deploy containers only on a compute resource of the Docker provider type. Therefore, when you attempt to view or create containers for the first time, Foreman prompts you to create a Docker compute resource. If you don't have a Docker host available, first create a container host as described in Procedure 1.1, “To Prepare a Container Host:”, then specify this host as a compute resource as described in Procedure 1.2, “To Create a Docker Compute Resource:”.
+
+
+**Procedure 1.1. To Prepare a Docker Host:**
```

Comment:
Is there the possibility to use `####` as headlining here with the current github-pages markdown? But I'm unsure if that's more or less confusing than the current "inline" numbering.